src/BAVRFieldController.cpp

```diff
@@ -269,32 +288,15 @@ void BAVRFieldController::interrupt(int pin)
 void BAVRFieldController::callback(char *topic, byte *payload, unsigned int length)
 {
 
-
   bool valid_message = false;
 
   clean_buffers(); // clean buffers before use
-  char uuid_initialization_topic[256];
```

Comment:
I removed these because we already subscribed to the topic for our node, so others are filtered out. No need to specify that here